view/frontend/templates/checkout/onepage/success/tesco_additional_info.phtml

```diff
@@ -0,0 +1,4 @@
+
+<p> <?= __('To finish payment, print the code below and bring it to make payment in any Tesco Lotus Store. Code is valid for the next <b>24 hours only</b>. After that time your order will be canceled.'); ?></p>
```

Comment:
@jacekstan just a little thing, there is a space after `<p>` tag. (maybe we don't need it?)